LiteCore/Database/LegacyAttachments.cc

```diff
@@ -59,6 +59,12 @@ namespace litecore { namespace legacy_attachments {
                 auto attachment = Value::asDict(i.value());
                 if (!attachment)
                     continue;
+                {
+                    // CBL-4944
+                    if (!attachment->get("length"_sl)) {
+                        C4Warn("Find attahcment with nil length in legacy_attachments::encodeStrippingOldMetaProperties: %s", attachment->toJSONString().c_str());
+                    }
```

Comment:
CBSE case has attachment length = 0 while it shouldn't be 0. The attachment length = 0 could be valid for empty blob but for the purpose of the investigation, we should log as well.